### PR TITLE
Better name splitting algo

### PR DESCRIPTION
**Pull request by suggoitanoshi**
I think this performs better when facing numerous cases, though I have yet to test for some edge cases.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,28 +6,6 @@ const cari = (str, dicari) => {
...
-      if ((tempStr + strArr[i] + '. ').length > 32
-        && (tempStr + initial).length <= 32) {
-        tempStr += initial;
-      } else if ((tempStr + strArr[i] + ' ').length <= 32) {
-        tempStr += strArr[i] + ' ';
-      }
-
-      ++i;
-    }
-  }
```
Comment (jspmarc) [inline index.js:9]: rip my code

File: index.js [hunk 2]
```diff
@@ -49,6 +27,24 @@ client.on('ready', () => {
...
+    i--;
+  }
+  thatName = thatName.join(' ');
+  if(currentLength > maxLength){
+    thatName = thatName.slice(0, maxLength-3)+'...';
+  }
+
+  return thatName;
+}
+
```
Comment (jspmarc) [inline index.js:47]: # nice

**Review by jspmarc — APPROVED** [review]
